lib/utilities/xml_http_request.js

```diff
@@ -50,12 +54,16 @@ Romo.define('Romo.XMLHttpRequest', function() {
       return true
     }
 
+    static successStatusCode(statusCode) {
+      return (statusCode >= 200 && statusCode < 300) || statusCode === 304
```

Comment:
@jcredding I had to bring this back b/c I forgot how XMLHttpRequest worked when I took it out. The `onload` event fire regardless of the server response code - all it cares about is did it get _any_ response from the server and if so `onload` fires. I guess that's why they didn't call it onsuccess eh?

Anyway, `Romo.XMLHttpRequest` binds with `onSuccess`, not `onLoad`, so we have to manually detect what "success" is. FYI.

Reply:
Gotcha, thanks!